doc/nrf/releases/release-notes-latest.rst

```diff
@@ -94,8 +94,47 @@ MCUboot
 sdk-mcuboot
 -----------
 
-The MCUboot fork in |NCS| contains all commits from the upstream MCUboot repository up to and including ``5a6e18148d``, plus some |NCS| specific additions.
-The list of the most important recent changes can be found in :ref:`ncs_release_notes_140`.
```

Comment:
@ru-fu Should I restore this line?

Reply:
No - we're listing changes since the last release, so we don't need that line in here anymore.